Api/Controllers/AuthController.cs

```diff
@@ -27,20 +27,24 @@ public class AuthController : ControllerBase
     private readonly AuthService _authService;
     private readonly IEmailService _emailService;
     private readonly UserManager<Account> _userManager;
+    private readonly string _confirmationTemplateId;
+    private readonly string _resetTemplateId;
 
     public AuthController(
         UserManager<Account> userManager,
         IAccountService accountService,
         IOptions<GoogleOptions> applicationSettings,
         AuthService authService,
-        IEmailService emailService, IConfiguration configuration)
+        IEmailService emailService, IConfiguration configuration, IOptions<SendGridSettings> sendGridSettings)
     {
         _userManager = userManager;
         _accountService = accountService;
         _authService = authService;
         _emailService = emailService;
         _applicationSettings = applicationSettings.Value;
-        _sendGridSenderEmail = configuration["SendGrid:SenderEmail"];
+        _sendGridSenderEmail = sendGridSettings.Value.SenderEmail;
```

Comment:
_sendGridSenderEmail = sendGridSettings.Value.SenderEmail;
        _confirmationTemplateId = sendGridSettings.Value.ConfirmationTemplateId;
        _resetTemplateId = sendGridSettings.Value.ResetTemplateId;
        |
        U can remove this lines